src/python/turicreate/extensions.py

```diff
@@ -115,24 +115,11 @@ def _setattr_wrapper(mod, key, value):
     if mod == _thismodule:
         setattr(_sys.modules[__name__], key, value)
 
-def _run_toolkit_function(fnname, arguments, args, kwargs):
+def _toolkit_function_pack_args(arguments, args, kwargs):
     """
-    Dispatches arguments to a toolkit function.
-
-    Parameters
-    ----------
-    fnname : string
```

Comment:
I think you probably just want to remove the `fnname` parameter from the docstring and leave the rest.

Reply:
Or perhaps I'm getting confused by how the diff is getting rendered and you haven't actually removed it. At any rate, you'll want this method to have a docstring.

Reply:
Must be the rendering.  It's got a docstring and that parameter.